OracleAccessManagement/README.md

```diff
@@ -0,0 +1,56 @@
+Oracle Access Management (OAM) on Docker
```

Comment:
As per Oracle policy, all documentation needs to adhere to Oracle documentation standards. This README should be reviewed by your documentation team.

Reply:
The documentation is still in progress. We will make the appropriate changes.

Reply:
This is taken care now

Reply:
Replace "on Docker" with "Container Image"